pycue/tests/api_test.py

```diff
@@ -189,7 +189,7 @@ def testGetJobs(self, getStubMock):
             jobs=job_pb2.JobSeq(jobs=[job_pb2.Job(name=TEST_JOB_NAME)]))
         getStubMock.return_value = stubMock
 
-        jobsByShow = opencue.api.getJobs(show=[TEST_SHOW_NAME], all=True)
+        jobsByShow = opencue.api.getJobs(show=[TEST_SHOW_NAME])
```

Comment:
It would be good to also add a test case for the new code -- to ensure an exception is thrown when there are incorrect parameters.

Reply:
Added two tests:
- test to check if all jobs get returned when criteria is not specified
- test to check if exception is raised when bad criteria is used